src/format.README.md

```diff
@@ -0,0 +1,10 @@
+# `format`
+
+Types and enums related to editorial formats.
+
+## Usage
+
+```js
+import type { Theme, Format } from '@guardian/libs';
+import { Pillar, Special, Design, Display } from '@guardian/libs';
```

Comment:
i think we need to think about these names here. i know they have big significance in editorial, but examples like this:

```js
import { Special } from '@guardian/libs';
import { Design } from '@guardian/libs';
```
need to convey more to non-editorial devs i think. esp when it pops up in a list of possible imports in autocomplete (and in a generic org-wide library).

@JamieB-gu @oliverlloyd does something like this make sense?

```js
import type { EditorialTheme, EditorialFormat } from '@guardian/libs';
import { EditorialPillar, EditorialSpecial, EditorialDesign, EditorialDisplay } from '@guardian/libs';
```

Reply:
I see your point. Given that `libs` is dept-wide and not Editorial-specific the names do lose some context. Perhaps this is the best solution 🤔.

Although it's tricky because by making it more specific for _non-consumers_ of these exports you're making it verbose/overly specific for the applications that _do_ consume them. I don't have an idea for how to solve that though 😔.

Given that these may not live in this library for long do we want to do the large find-and-replace PRs on DCR, AR and IR to accommodate this change?

Apologies @sndrs none of what I've said is particularly conclusive, I've just added some more questions 🙈

cc @gtrufitt

Reply:
Yeah, `import { Special } from '@guardian/libs';` lacks a lot of context outside of our bubble 🤔 

`EditorialSpecial` is explicit but also a bit clunky. You'd probably end up  `import { EditorialSpecial as Special } from '@guardian/libs';` which I guess is fine but also a bit meh.

I'm guessing not but Is there an option to use a folder, like

```ts
import { Special } from '@guardian/libs/editorial';
```
?

Another option is to do what DCR does (in about 80% of places) and use the `Type` suffix, like

```ts
import { SpecialType, PillarType } from '@guardian/libs';
```

Reply:
the folder thing is an option but i'd rather avoid restructuring libs in order to accommodate importing ambiguous exports from `types`. e.g. we could potentially end up exporting two things with the same from different places which is even more ambiguity. 

how do you feel about these instead?

```ts
import type { ArticleTheme, ArticleFormat } from '@guardian/libs';
import { ArticlePillar, ArticleSpecial, ArticleDesign, ArticleDisplay } from '@guardian/libs';
```
it has the advantage of not implying that only certain teams should them and `ArticleDesign.MatchReport` reads clearly to me?

another option: if this is still unacceptably verbose, perhaps that's a sign these things shouldn't live in here?

Reply:
I think I'd actually prefer `EditorialDesign` to `ArticleDesign`, because `Article` _is_ a `Design`, so you'll end up with `ArticleDesign.Article` 🙈.

> another option: if this is still unacceptably verbose, perhaps that's a sign these things shouldn't live in here?

That's reasonable. It's possible that they don't. They're specifically Editorial concepts and this is meant to be a dept-wide repo. They're only being put here for temporary convenience until we can get them into CAPI. Maybe instead we should just not deprecate `@guardian/types` until that migration to CAPI has finished?

Reply:
It also occurs to me that if we _do_ merge the DCR and AR repos then having `Format` in a library becomes unnecessary. We can just put it in a shared folder in that repo.

Reply:
I mean, really, `Article` should be `News` or `Standard` and then `ArticleDesign` would work really nicely 😄

Reply:
ah right that's interesting. so it sounds as though something like `EditorialContentDesign.Article` is probably an _accurate_, unambiguous name (or something like it), but the amount of context you need to pack in to that name perhaps means it's not generic enough to live in here (whereas living in CAPI provides that context implicitly)

Reply:
> I mean, really, `Article` should be `News` or `Standard`

It probably should be `Standard`, you're right Oliver 🙄.

I don't think it should be `News` because a) it's not always used for news pieces, and b) `News` is a pillar 🙂.

Reply:
Just to clarify, is everyone happy for these values to change to

```ts
import type { ArticleTheme, ArticleFormat } from '@guardian/libs';
import { ArticlePillar, ArticleSpecial, ArticleDesign, ArticleDisplay } from '@guardian/libs';
```

and `Design.Article` to become `ArticleDesign.Standard` or do we close this PR and figure out a better place for `Format` to live?

Reply:
I'm happy with `ArticleTheme` etc. 👍 

Maybe the move from `Article` > `Standard` could be a separate PR?

Reply:
But also 👍 for `Standard`